theme/style.css

```diff
@@ -15,4 +15,10 @@
   text-align: center;
 }
 
+h3 {
```

Comment:
This is too broad a selector... please make sure that your changes only affect your square!